docs/requirements.txt

```
@@ -5,5 +5,5 @@
 # the project.
 
 sphinx >=3.0, <5.0
-sphinx-rtd-theme >=0.5, <1.0
+sphinx-rtd-theme >=0.5, <2.0
```

Remove minimum requirement
```suggestion
sphinx-rtd-theme <2.0
```